.github/workflows/dirgraph.yml

```diff
@@ -0,0 +1,31 @@
+name: Test Dirgraph
```

Comment:
I would change this to "Test visualization application" or something slightly more descriptive